backend/converter.py

```diff
@@ -349,7 +349,7 @@ def extract_komunalni_candidates(
         candidate = Candidate(
             id=candidate_id,
             num=int(row["id"]),
-            name=str(row["name"]),
```

Comment:
This should be handled on FE by rendering the proper attribute.

Logic should be:
```
if candidate.type == PERSON then show parties[0].name
if candidate.type == PARTY then show parties[0].short_name
```

Reply:
There has to be 2 changes:
1. Add that attribute - https://github.com/KohoVolit/volebni-kalkulacka-2022/pull/144/files
2. Use that attribute - https://github.com/KohoVolit/volebni-kalkulacka-2022/pull/143